Classes/Domain/Model/Holder.php

```diff
@@ -18,60 +18,27 @@
  */
 class Holder extends AbstractEntity
 {
-    /**
-     * @var string
-     */
-    protected $title = '';
-
-    /**
-     * @var string
-     */
-    protected $contactPerson = '';
-
-    /**
-     * @var string
-     */
-    protected $street = '';
-
-    /**
-     * @var string
-     */
-    protected $houseNumber = '';
-
-    /**
-     * @var string
-     */
-    protected $zip = '';
-
-    /**
-     * @var string
-     */
-    protected $city = '';
-
-    /**
-     * @var string
-     */
-    protected $telephone = '';
-
-    /**
-     * @var string
-     */
-    protected $fax = '';
-
-    /**
-     * @var string
-     */
-    protected $email = '';
-
-    /**
-     * @var string
-     */
-    protected $website = '';
-
-    /**
-     * @var string
-     */
-    protected $logo = '';
+    protected string $title = '';
+
+    protected string $contactPerson = '';
+
+    protected string $street = '';
+
+    protected string $houseNumber = '';
+
+    protected string $zip = '';
+
+    protected string $city = '';
+
+    protected string $telephone = '';
+
+    protected string $fax = '';
+
+    protected string $email = '';
+
+    protected string $website = '';
+
+    protected FileReference $logo;
```

Comment:
Please use ObjectStorage. I know, the UpgradeWizard have to be updated, too. But: The table `sys_file_reference` is a `MM` table with `uid_local` and `uid_foreign`. Only because this table has additional fields it got an additional `uid` column. Please do not relate to this `uid`. Never.